services/contentapi/main.py

```diff
@@ -1,10 +1,25 @@
 from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
 from pydantic import BaseModel
+from pathlib import Path
+
+DEFAULT_CONTENT_TEMPLATE = "<div><p>This is content for ID {}</p>" \
+    "<p>Math: \\( \\frac{{1}}{{2}} \\)</p></div>"
+
+
+HTML_DATA_PATH = "/html"
+
 
 app = FastAPI(
     title="RAISE Content API"
 )
 
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_methods=["GET"]
+)
+
```

Comment:
Here's a nice [primer on CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) that's worth a read to understand what it is. Experimentally, you can remove the CORS changes in your local environment (then `docker-compose build` --> `docker-compose up -d`). Then you'll see content stops rendering and if you open the dev tools in your browser you should see CORS errors because the site is at `localhost:8000` but we're making requests to `localhost:8800`.

In our case, the fix is to use Middleware with FastAPI, and it just so happens the library provides a built-in middleware to deal with CORS. Some useful links:

- [FastAPI Middleware](https://fastapi.tiangolo.com/tutorial/middleware/)
- [FastAPI CORS Middlware](https://fastapi.tiangolo.com/tutorial/cors/)]